### ISSUE TITLE
mobileone-s1 prediction scores are low

### ISSUE THREAD

**Issue by Phil-Lin** [open-mmlab/mmclassification#1845]
### 分支

main 分支 (mmpretrain 版本)

### 描述该错误

hello，I pre-trained the model mobileone-s1 in Imagenet 1k，and get TOP ACC 75.2 in testSet of Imagenet 1k and loss during training is normal.  I deploy the onnx model by mmdeploy for my pretrain checkpoint, and the prediction scores of model are low, prediction scores are low.
![car](https://github.com/open-mmlab/mmpretrain/assets/52906245/23e82c92-f797-481e-b07d-4d2280114d46)
![dog](https://github.com/open-mmlab/mmpretrain/assets/52906245/5d0e5303-486d-4712-96b0-5ee8ca8b19ea)

### 环境信息

{'sys.platform': 'linux',
 'Python': '3.8.18 (default, Sep 11 2023, 13:40:15) [GCC 11.2.0]',
 'CUDA available': True,
 'numpy_random_seed': 2147483648,
 'GPU 0,1,2,3,4,5,6,7': 'NVIDIA GeForce RTX 4090',
 'CUDA_HOME': '/usr/local/cuda:',
 'GCC': 'gcc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0',
 'PyTorch': '1.10.1+cu111',
 'TorchVision': '0.11.2+cu111',
 'OpenCV': '4.8.1',
 'MMEngine': '0.9.0',
 'MMCV': '2.0.1',
 'MMPreTrain': '1.1.0+'}
